src/scheduler/tasks/ocpi/OCPIPushLocationsTask.ts

```diff
@@ -67,6 +67,9 @@ export default class OCPIPushLocationsTask extends SchedulerTask {
         });
         // Build OCPI Client
         const ocpiClient = await OCPIClientFactory.getCpoOcpiClient(tenant, ocpiEndpoint);
+        // TODO: Investigate if the PushLocationsTask only pushes the EVSE Statuses and not all EVSEs / Locations
+        // CpoOcpiClient has no method sendLocations or pushLocation
+        // sendEVSEStatuses() -> patchEVSEStatus() only contains the EVSE status in the payload and no location
         // Send EVSE statuses
         const sendResult = await ocpiClient.sendEVSEStatuses(!Utils.isUndefined(config.processAllEVSEs) ? config.processAllEVSEs : false);
```

Comment:
pushLocationsTask only pushes the EVSE Statuses

Reply:
The locations handling is in the ocpiClient.checkLocations() and is used in the matching task OCPICheckLocationsTask. 
That task is just not correctly named and shall be renamed to OCPIPushEVSESStatusesTask.ts. 

Could you please do the renaming and audit all code paths using ocpiClient.sendEVSEStatuses() to ensure it's used with the right naming? (update also configuration files) 

The PUT on the location endpoint to add/update locations is not implemented and currently Gireve fetch the locations on a regular basis by using the CPO endpoint on our OCPI server.

Reply:
done. renamed the objects from pushLocation to pushEVSEStatuses

Reply:
Thanks. 

I think you can undraft the PR and put in the review pipeline.